sass/directives/_tables.scss

```diff
@@ -38,76 +38,146 @@
   border-top-right-radius: $table-header-corner-radius;
 }
 
-@mixin table--product($table-highlight-z-index: 1, $table-caption-z-index: 2, $table-col-width: 25%) {
-  @include table--row-column-headers($table-col-width, $table-caption-z-index);
-  overflow: hidden;
+
+// Compare Charts
+
+@mixin table--row-column-headers($table-caption-z-index: 2) {
+  border-collapse: collapse;
+  border-spacing: $table-border-spacing;
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    position: relative;
+    z-index: $table-caption-z-index;
+    background: $panel-primary;
+    text-align: left;
+  }
 
   th, td {
-    // Column highlight
-    @include table__column-highlight('', $table-highlight-z-index);
+    border: $base-border;
   }
 
-  // Remove column highlight
-  th[scope='row'],
-  thead td {
-    @include table__column-highlight(none, $table-highlight-z-index);
+  th {
+    font-weight: $font-weight-bold;
+    text-transform: uppercase;
+    font-size: 0.9375rem;
   }
 
-  thead {
-    th {
-      color: white;
-      font-weight: $font-weight-bold;
-    }
+  thead th:first-of-type {
+    border: 0;
+    background-color: transparent;
 
-    th:nth-of-type(1n) {
-      background-color: $table-column-heading-1;
+    @media only screen and (max-width: $medium-screen-max) {
+      width: 14ch;
     }
+  }
 
-    th:nth-of-type(2n) {
-      background-color: $table-column-heading-2;
-    }
+  th[scope='col'] {
+    padding: $base-spacing-medium;
+    color: white;
+    text-align: center;
+    position: relative;
+    background-clip: padding-box;
+  }
 
-    th:nth-of-type(3n) {
-      background-color: $table-column-heading-3;
+  th[scope='row'] {
+    padding: $table--row-column-headers--row-top-bottom-padding $base-spacing-medium;
```

Comment:
yeah it is a long variable name :( but the name reflects a table > with the row-and-column-headers style (as opposed to the other type of table on the [ecom page](https://hiring.careerbuilder.com/promotions) which just has column headers:

![image](https://user-images.githubusercontent.com/2359538/26988712-02bc81a0-4d16-11e7-940e-f490555bf8e7.png)

that table above uses mixins that are generically named `table` and those need to be refactored to use a modifier as well, so it's clear what each set of mixins is used for

and `row-top-bottom-padding` reflects that this padding isn't what's used on the top and bottom of the column headers, it's only used on the top and bottom of rows, including the `th`s and the `td`s within those rows